internal/acctest/acctest.go

```diff
@@ -220,10 +220,12 @@ func PreCheck(t *testing.T) {
 		region := Region()
 		os.Setenv(envvar.DefaultRegion, region)
 
-		err := sdkdiag.DiagnosticsError(Provider.Configure(context.Background(), terraform.NewResourceConfigRaw(nil)))
+		// TODO: take `ctx` as a parameter instead
+		ctx := Context(t)
```

Comment:
Arriving in https://github.com/hashicorp/terraform-provider-aws/pull/29657 😄.
I'll rebase and merge it once things have quietened down after the release.